src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -1,34 +1,96 @@
 import React from 'react';
 import styled from 'styled-components';
-import { shape, string } from 'prop-types';
+import { shape, string, node } from 'prop-types';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/dist/spacings';
 import { GridItemConstrainedMedium } from '#lib/styledGrid';
 
+const KEYLINE_WIDTH = '0.125rem';
+
 const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const AmpIframe = ({ children, className, width, height, src }) => (
+  <amp-iframe
+    class={className}
+    width={width}
+    height={height}
+    layout="responsive"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
+    resizable
+    src={src}
+  >
+    {children}
+  </amp-iframe>
+);
+
+const StyledAmpIframe = styled(AmpIframe)`
+  > [overflow] {
```

Comment:
What does `[overflow]` do here, is that pure CSS or styled components?

Reply:
It's an attribute selector – https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Attribute_selectors
It's used in the AMP stylesheet to style the overflow element and we use it to update the styles with the same specificity.

Reply:
Yes, it is pure CSS. It is selecting the overflow element where styles have been injected by the amp-iframe and then the styles set in `StyledAmpIframe` override these styles set by the amp-iframe. Previously I used the `&&&` which was increasing specificity by adding multiple classnames, however @simonsinclair showed me that using this child element selector `> [overflow]`, is a cleaner approach, as now we are overriding the styles on the element where they are set.

Reply:
Thanks for explaining guys, makes sense 😄